examples/hybrid/callbacks.jl

```diff
@@ -272,3 +285,18 @@ function save_to_disk_func(integrator)
     Base.close(hdfwriter)
     return nothing
 end
+
+function save_restart_func(integrator)
+    (; t, u, p) = integrator
+    (; output_dir) = p.simulation
+    Y = u
+    day = floor(Int, t / (60 * 60 * 24))
+    sec = Int(mod(t, 3600 * 24))
+    @info "Saving restart file to HDF5 file on day $day second $sec"
+    output_file = joinpath(output_dir, "restart_day$day.$sec.hdf5")
```

Comment:
How about `joinpath(output_dir, "restart", "day$day.$sec.hdf5")`? I prefer to put all restart files into one folder.

Reply:
Yeah, sounds like a good idea